query.py

```diff
@@ -0,0 +1,394 @@
+import json
+import requests
+import networkx as nx
+
+
+class Query:
+    def __init__(
+        self,
+        service_base_url: str,
+        CapabilityStatement_path: str = None,
+    ) -> None:
+        self.base = service_base_url
+        self.possible_references = self._get_rev_include_possibilitie(
+            CapabilityStatement_path
+        )
+        """Instantiate the class and create the query object
+
+        Arguments:
+            service_base_url {str} -- the Service Base URL
+            CapabilityStatement_path {str} -- path to the json file that
+            contains a resource of type CapabilityStatement
+        """
+
+        # to represent the relationships (references) between resources
+        self.rscs_graph = nx.DiGraph()
```

Comment:
What does `rscs` mean?

Reply:
I'd prefer if we use the full word, indeed. We don't gain much with the abreviation 🙂

Reply:
Done :)